### ISSUE TITLE
UI (b32): Size / alignment discrepancy of RP logo in menu vs. app main view 

### ISSUE THREAD

**Issue by bradtchapman** [BillGoldsmith/iOS-bugfest#73]
### Description of issue

The addition of the Support thermometer has caused the RP logo to shrink considerably.  But when the menu is opened, the same logo is nearly doubled in size and shifted (the shift was previously reported in another issue).

Logo should be the same size when the menu is opened or closed.

### Demo

See attached screen recording (cropped to relevant section of screen).

https://user-images.githubusercontent.com/32527656/218930612-370985be-9ff8-4afa-b1c0-601da6063cac.MOV

**Comment by RPNate1** [comment]
This actually doesn't bother me. On my device (11 Pro) I only have the headphones displayed until the menu pops open with the logo. The headphones don't change in size. I think this may be a balancing act across screen sizes. If it is, We'll have to live with slightly different logo sizes.

**Comment by bradtchapman** [comment]
When you fix #66 , the solution for this one becomes clearer.

**Comment by RPNate1** [comment]
You're right. It became clearer when I considered that one. I take back my above comment.

**Comment by donshefer** [comment]
fixed targeting build 33

**Comment by bradtchapman** [comment]
Better ... but not perfect yet.  Checked on iPhone 12 Pro.   Will double check other devices this afternoon.  Thanks for all you do.
